include/DataFrame/Utils/Concepts.h

```diff
@@ -128,7 +128,7 @@ concept hashable_equal = hashable<T> && std::equality_comparable<T>;
 template<typename T>
 concept hashable_stringable = requires(const std::remove_reference_t<T> &a,
                                        std::stringstream &ss)  {
-    hashable<T>;
+    requires hashable<T>;
```

Comment:
are you sure `requires` is required here?

Reply:
Yes GCC complains if requires is missing. I'll remove and show the output.  Hold on.

Reply:
`Concepts.h:131:5: warning: testing if a concept-id is a valid expression; add 'requires' to check satisfaction [-Wmissing-requires]
  131 |     hashable<T>;
      |     ^
      |     requires 
`

Reply:
So by adding `requires` the warning is satisfied.